chat-plugins/scavengers.js

```diff
@@ -1006,7 +1006,7 @@ let commands = {
 	 */
 	queue: function (target, room, user) {
 		if (room.id !== 'scavengers') return this.errorReply("This command can only be used in the scavengers room.");
-		if (!target) return this.parse("/scavengers viewqueue");
+		if (!target) return this.parse(this.message.replace('queue', 'viewqueue'));
```

Comment:
i think the standard for this is ``this.parse(`${this.cmdToken}scavengers viewqueue`)``, see https://github.com/Zarel/Pokemon-Showdown/blob/master/chat-commands.js#L170